src/app/search/page.module.css

```
@@ -0,0 +1,85 @@
+  background-color: #000000;
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.list_container {
+  --panel-horizontal-padding: 24px;
+  --panel-content-width: 640px;
+  position: relative;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)